### PR TITLE
Improved referendum display

### PR DESCRIPTION
**Pull request by dillchen**
## Description

Provides fixes on the proposals and councils pages for handling certain text overflows. Cases like the ones below have been solved.

**Before**
image
image
image

**After:**
image
image
image

### CODE REVIEW

File: server/routes/upgradeMember.ts [hunk 1]
```diff
@@ -28,7 +33,7 @@ const upgradeMember = async (models, req: UserRequest, res: Response, next: Next
   let member = await models.Role.findOne({
     where: {
       ...chainOrCommObj,
-      address_id: memberAddress.id,
+      address_id: memberAddress.address,
```
Comment (raykyri) [inline server/routes/upgradeMember.ts:32]: Why is this changed?

Comment (dillchen) [reply server/routes/upgradeMember.ts:32]: I'm not quite sure why that change is included. Sorry about this

Comment (dillchen) [reply server/routes/upgradeMember.ts:32]: reverted in latest commit

Comment (raykyri) [reply server/routes/upgradeMember.ts:32]: This still looks off to me, it looks like the address_id at the top was reverted but not this one?
